print-battleground-state-changes

```diff
@@ -275,14 +281,14 @@ print(tabulate([
     ["Prettier web version:", "https://alex.github.io/nyt-2020-election-scraper/battleground-state-changes.html"],
 ]))
 for (state, timestamped_results) in summarized.items():
-    print(f'\n{state}:')
+    print(f'\n{state}: Total Votes: ({timestamped_results[1][1]}: {timestamped_results[1][3]}, {timestamped_results[1][2]}: {timestamped_results[1][4]})')
     print(tabulate([string_summary(summary) for summary in timestamped_results]))
 
     # 'Alaska (3)' -> 'alaska', 'North Carolina (15)' -> 'north-carolina'
     state_slug = state.split('(')[0].strip().replace(' ', '-').lower()
     html_chunks.append(f"<table id='{state_slug}' class='table table-bordered'>")
-    html_chunks.append(html_write_state_head(state))
     for summary in timestamped_results:
+        html_chunks.append(html_write_state_head(state, summary))
```

Comment:
Writing the thead for every row seems like a lot? Can you post a screenshot of what this looks like

Reply:
Good spot! It looks wonky, let me refactor that :)

Reply:
done. and this is what it looks like now. <img width="1666" alt="Screen Shot 2020-11-05 at 7 35 43 PM" src="https://user-images.githubusercontent.com/10540830/98311750-27535a80-1f9e-11eb-8fb7-bc11a3df8885.png">

Reply:
should I also make another commit emphasizing that the parenthetical is for EV?